app/src/main/java/com/ivy/wallet/RootActivity.kt

```diff
@@ -73,6 +77,7 @@ class RootActivity : AppCompatActivity(), RootScreen {
     private lateinit var onFileOpened: (fileUri: Uri) -> Unit
 
     private val viewModel: RootViewModel by viewModels()
+    private val settingsViewModel: SettingsViewModel by viewModels()
```

Comment:
Nah, that should be done inside the `SettingsViewModel` and its module. Don't add stuff to `:app`